akka-actor/src/main/scala/akka/io/TcpListener.scala

```diff
@@ -70,8 +70,13 @@ private[io] class TcpListener(
       ret
     } catch {
       case NonFatal(e) =>
-        bindCommander ! bind.failureMessage.withCause(e)
-        log.error(e, "Bind failed for TCP channel on endpoint [{}]", bind.localAddress)
+        val exception = if (e.isInstanceOf[java.net.BindException]) {
+          new java.net.BindException(s"[${bind.localAddress}] ${e.getMessage}")
```

Comment:
We loose the original stack trace here. Shall we grab it from the original exception with `newException.setStackTrace(e.getStackTrace)`?

Reply:
That's right!